fatips/1.md

```diff
@@ -1,226 +1,171 @@
-| FATIP | Title                       | Status | Category       | Author                             | Requires | Created    |
-| ----- | --------------------------- | ------ | -------------- | ---------------------------------- | -------- | ---------- |
-| 1     | Non-Fungible Token Standard | Draft  | Token Standard | Devon Katz \<<devonk@dbgrow.com>\> | -        | 2018-07-27 |
+| FATIP | Title                       | Status   | Category       | Author                             | Created    |
+| ----- | --------------------------- | -------- | -------------- | ---------------------------------- | ---------- |
+| 1     | Non-Fungible Token Standard | Accepted | Token Standard | Devon Katz \<<devonk@dbgrow.com>\> | 2018-07-27 |
 
 
 
 # Overview
 
-This document describes the functionality, datastructures, validation rules of
-the **FAT-1** token standard: A non-fungible data only token standard built on
-Factom chains and entries. It's functionality is most alike the ERC-721 token
-standard of Ethereum.
+This document describes the **FAT-1** token standard: A non-fungible (NF) token standard. This this standard allows for token systems supporting unique, indivisible tokens issued in whole units. It is most equivalent to Ethereum's ERC-721 token standard.
+
 
 
 # Motivation
 
-Extend upon the FAT-0 standard by offering non-fungible token functionality for
-unique, indivisible, and traceable assets.
+Offer an efficient non-fungible token system, allowing attachment of arbitray metadata to issued tokens. Allow unique, indivisible, and traceable assets on top of FAT.
```

Comment:
`on top of FAT`? Typo? FAT is a layer on top of Factom, but FAT-1 is not on top of FAT.